src/powers.cc

```diff
@@ -195,11 +194,6 @@ static void power_activate(int power)
 			}
 		}
 		break;
-	case PWR_LAY_TRAP:
```

Comment:
I wonder... are there any remaining entries in any *.txt files which "point" to this case? (Searching for the number assigned to PWR_LAY_TRAP will probably be enough to tell.)